precompiles/conviction-voting/src/lib.rs

```diff
@@ -165,9 +133,13 @@ where
 		Self::vote(
 			handle,
 			poll_index,
-			VoteDirection::Yes,
-			vote_amount,
-			conviction,
+			AccountVote::Standard {
+				vote: Vote {
+					aye: true,
+					conviction: Self::u8_to_conviction(conviction)?,
```

Comment:
Any reason this doesn't have `in_field`?

```suggestion
					conviction: Self::u8_to_conviction(conviction).in_field("conviction")?,
```